installers/docker-compose/legend-with-remote-gitlab/README.md

```diff
@@ -0,0 +1,60 @@
+# Docker Compose for Legend with remote instance of Gitlab
```

Comment:
Please restructure the instructions such that there is a linear flow of steps. In its current form, the user has to go back and forth between the Gitlab OAuth configuration steps and this is not clear.

Reply:
Gotcha, I have updated the README files with more detailed steps.